dashboard/lib/tasks/seed.rake

```diff
@@ -64,7 +64,7 @@ namespace :seed do
 
   # detect changes to dsldefined level files
   # LevelGroup must be last here so that LevelGroups are seeded after all levels that they can contain
-  DSL_TYPES = %w(TextMatch ContractMatch External Match Multi LevelGroup)
+  DSL_TYPES = %w(TextMatch ContractMatch External Match Multi EvaluationMulti LevelGroup)
```

Comment:
Unfortunately it depends on the models being loaded already.  There is [`initializers/load_all_models.rb`](https://github.com/code-dot-org/code-dot-org/blob/26f099b7c4797c28f317bef05fd624dcf8c31198/dashboard/config/initializers/load_all_models.rb) but naturally it doesn't contain all models 😄.